x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/apikey/RestCreateCrossClusterApiKeyAction.java

```diff
@@ -79,4 +81,16 @@ protected RestChannelConsumer innerPrepareRequest(final RestRequest request, fin
             new RestToXContentListener<>(channel)
         );
     }
+
+    @Override
+    protected Exception checkFeatureAvailable(RestRequest request) {
+        final Exception failedFeature = super.checkFeatureAvailable(request);
+        if (failedFeature != null) {
+            return failedFeature;
+        } else if (ADVANCED_REMOTE_CLUSTER_SECURITY_FEATURE.checkWithoutTracking(licenseState)) {
```

Comment:
I went back and forth with whether this should `check` or `checkWithoutTracking`. At the end, I went with `checkWithoutTracking` because it is more consistent to track actual usage of cross-cluster access. Otherwise, the stats can be skewed by just playing with these APIs.